hexrdgui/image_canvas.py

```diff
@@ -710,7 +737,7 @@ def show_saturation(self):
             return
 
         # Use the unscaled image data to determine saturation
-        images_dict = self.unscaled_image_dict
+        images_dict = self.unscaled_display_image_dict
```

Comment:
@psavery This was one place where I wasn't 100% certain - I am pretty sure we should be using the display image here but I'd like to make sure that's correct

Reply:
I think for computing the saturation, we should leave it as the `self.unscaled_image_dict`.

But the saturation is only computed in the raw view anyways, and there's no difference between the display and computed images in the raw view.

Let's just change it back to `self.unscaled_image_dict`